lessons/13-server-rendering/server.js

```diff
@@ -7,7 +7,7 @@ var app = express()
 app.use(compression())
 
 // serve our static stuff like index.css
-app.use(express.static(path.join(__dirname, 'public')))
+app.use(express.static(__dirname, 'public'))
```

Comment:
This was correct previously. The other examples have it:

* https://github.com/reactjs/react-router-tutorial/blob/master/lessons/12-navigating/server.js#L14
* https://github.com/reactjs/react-router-tutorial/blob/master/lessons/14-whats-next/server.js#L14